src/features/elementfeature.jl

```diff
@@ -74,27 +94,48 @@ end
 
 function (f::ElementFeatureDescriptor)(a::AbstractAtoms)
     @assert all([el in encodable_elements(f) for el in a.elements]) "Feature $(f.name) cannot encode some element(s) in this structure!"
+    f.encoder_decoder(f, a, ENCODE)
+end
+
+
+function (ed::DummyED)(e::ElementFeatureDescriptor, a::AbstractAtoms, e_or_d::EncodeOrDecode)
+    if e_or_d == ENCODE
+        ed.encode_f(e, a, ed.nbins, ed.logspaced)
+    else
+        ed.decode(e, a, ed.nbins, ed.logspaced)
+    end
+end
+
+decode(f::ElementFeatureDescriptor, encoded_feature) =
```

Comment:
can we add a function `encode(f::ElementFeatureDescriptor, a::AbstractAtoms)` that does the same thing as the "callable syntax" just for symmetry with this one? I could imagine a user trying to do that and being frustrated that they couldn't...

Reply:
Resolved in b5e5406.